src/engraving/accessibility/accessibleitem.cpp

```diff
@@ -187,11 +189,25 @@ QString AccessibleItem::accessibleName() const
     QString commandInfo = root ? root->commandInfo() : "";
     QString staffInfo = root ? root->staffInfo() : "";
     QString barsAndBeats = m_element->formatBarsAndBeats();
+    Selection selection = m_element->score()->selection();
+    QString selectionText = "";
+
+    if (selection.isRange() && selection.endSegment()) {
+        Segment* prevSeg = selection.endSegment()->prev1(SegmentType::ChordRest);
+        if (m_element->tick() != selection.tickStart() || m_element->tick() != prevSeg->tick()) {
+            if (m_element->tick() == selection.tickStart()) {
+                selectionText = "Selection begins";
+            } else if (prevSeg && m_element->tick() == prevSeg->tick()) {
+                selectionText = "Selection ends";
+            }
+        }
+    }
 
     barsAndBeats.remove(u';'); // Too many pauses in speech
 
-    QString name = QString("%1%2%3%4%5")
+    QString name = QString("%1%2%3%4%5%6")
                    .arg(!commandInfo.isEmpty() ? (commandInfo + "; ") : "")
+                   .arg(selectionText)
```

Comment:
I think you need to do this:
```suggestion
                   .arg(!selectionText.isEmpty() ? (selectionText + "; ") : "")
```
otherwise it will be glued together with the next word